circle.yml

```diff
@@ -19,7 +19,7 @@ dependencies:
 
 test:
   override:
-    - ./gradlew clean :core:install :core:verify :mapzen-android-sdk:verify :mapzen-places-api:verify --configure-on-demand
+    - ./gradlew clean :core:install :core:verify :mapzen-android-sdk:verify :mapzen-places-api:verify --configure-on-demand -PdisablePreDex
```

Comment:
Using `-PdisablePreDex` was recommended in [this Circle CI issue](https://discuss.circleci.com/t/java-lang-outofmemoryerror-permgen-space-in-android-gradle-build/3652).